jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/ServletContextHandlerTest.java

```diff
@@ -2368,4 +2373,100 @@ public void init() throws ServletException
         assertThat(response, containsString("200 OK"));
         assertThat(response, containsString("/three"));
     }
+
+    static class CookieTweakResponseApi extends ServletApiResponse
+    {
+        CookieTweakResponseApi(ServletApiResponse response)
+        {
+            super(response.getResponse());
+        }
+
+        @Override
+        public void addCookie(Cookie cookie)
+        {
+            // Let's set SameSite to STRICT
+            // But we cannot use Servlet jakarta.servlet.http.Cookie.setComment(String) technique of old
+            // as the Comment field is always null and is slated for removal.
```

Comment:
Let's remove the comment handling code if it is no longer usable.... or do we still need it for ee8/ee9?

Reply:
I'll remove from ee10 only, as comment is only deprecated there.

Reply:
Actually, this is turning out to be a bit change in itself, I'll do the minimal here in this PR and follow it up with a the larger change in a different PR. (to avoid omnibus PRs)